### PR TITLE
Replace SingleAutocompleteSelectField with Combobox

### PR DESCRIPTION
**Pull request by poulch**
## What type of PR is this?
- [ ] 💅 Refactor
- [ ] 🌟 Feature
- [ ] 🔥 Bug Fix
- [ ] 🔩 Maintenance
- [ ] 🛠 Workflow CI/CD changes

## Related Issues or Documents

- closes #

## Have you written tests?
- [ ] Yes!
- [ ] No... here is why: _Writing tests are mandatory, please replace this text with why test are not included in this PR_

### CODE REVIEW

File: src/components/SingleAutocompleteSelectField/SingleAutocompleteSelectField.tsx [hunk 1]
```diff
@@ -41,7 +40,7 @@ export interface SingleAutocompleteSelectFieldProps extends Partial<FetchMorePro
   fetchChoices?: (value: string) => void;
   onChange: (event: React.ChangeEvent<any>) => void;
   fetchOnFocus?: boolean;
-  FormHelperTextProps?: ExtendedFormHelperTextProps;
+  FormHelperTextProps?: any;
```
Comment (poulch) [inline src/components/SingleAutocompleteSelectField/SingleAutocompleteSelectField.tsx:43]: This component will be removed I didn't want to play with typing

File: .changeset/few-crabs-guess.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Tou can now see all options in the autocomplete select field
```
Comment (Cloud11PL) [inline .changeset/few-crabs-guess.md:5]:
```suggestion
You can now see all options in the autocomplete select field
```

File: src/components/AddressEdit/AddressEdit.tsx [hunk 3]
```diff
@@ -222,46 +222,48 @@ const AddressEdit: React.FC<AddressEditProps> = props => {
 
       <div className={classes.root}>
         <div>
-          <SingleAutocompleteSelectField
-            disabled={disabled}
-            autocomplete="new-password"
+          <Combobox
             data-test-id="address-edit-country-select-field"
-            displayValue={countryDisplayValue}
+            autoComplete="new-password"
```
Comment (Cloud11PL) [inline src/components/AddressEdit/AddressEdit.tsx:227]: Suggestion: I remember this was a hack for browser keeping customers' addresses. Let's change this to "off" (there are multiple in this component)

File: src/components/AddressEdit/AddressEdit.tsx [hunk 4]
```diff
@@ -222,46 +222,48 @@ const AddressEdit: React.FC<AddressEditProps> = props => {
...
         </div>
         <div>
           {isFieldAllowed(PossibleFormFields.COUNTRY_AREA) && (
-            <SingleAutocompleteSelectField
-              disabled={disabled}
-              autocomplete="new-password"
+            <Combobox
               data-test-id="address-edit-country-area-field"
-              displayValue={getDisplayValue(data.countryArea)}
+              autoComplete="new-password"
```
Comment (Cloud11PL) [inline src/components/AddressEdit/AddressEdit.tsx:250]: Suggestion: Same as above.

File: src/components/CompanyAddressInput/CompanyAddressForm.tsx [hunk 5]
```diff
@@ -170,45 +169,47 @@ const CompanyAddressForm: React.FC<CompanyAddressFormProps> = props => {
       </Grid>
       <FormSpacer />
       <Grid>
-        <SingleAutocompleteSelectField
+        <Combobox
           data-test-id="address-edit-country-select-field"
+          autoComplete="new-password"
```
Comment (Cloud11PL) [inline src/components/CompanyAddressInput/CompanyAddressForm.tsx:174]: Suggestion: Same as above

**Review by Cloud11PL — APPROVED** [review]

**Review by krzysztofzuraw — APPROVED** [review]

File: .changeset/few-crabs-guess.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You can now see all options in the autocomplete select field
```
Comment (andrzejewsky) [inline .changeset/few-crabs-guess.md:5]: - previously i couldn't see all?
- in the autocomplete select field - all autocompletes? which ones?

**Pull request merged**